newIDE/app/src/MainFrame/EditorContainers/BaseEditor.js

```diff
@@ -94,6 +95,12 @@ export type RenderEditorContainerProps = {|
   askToCloseProject: () => Promise<boolean>,
   closeProject: () => Promise<void>,
 
+  // Games
+  games: ?Array<Game>,
+  fetchGames: () => Promise<void>,
+  onGameUpdated: (game: Game) => void,
+  gamesFetchingError: ?Error,
```

Comment:
Could this be passed as `gamesList: GamesList` type?
I'm afraid of these 4 properties becoming 5, then 6, then 7.